sdk/communication/azure-communication-administration/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.0.0b4 (Unreleased)
 
 ### Breaking Changes
+- CommunicationIdentityClient is moved to a new package: "azure.communication.identity"
```

Comment:
we also need this package to be added to the ci.yml file for letting this run in the pipeline too. But it make sense to have another PR for this part.

Reply:
I have added the identity package to ci.yml file.